webpack.config.js

```diff
@@ -95,6 +95,8 @@ if ( CALYPSO_ENV === 'desktop' || CALYPSO_ENV === 'desktop-mac-app-store' ) {
 	// no chunks or dll here, just one big file for the desktop app
 	webpackConfig.output.filename = '[name].js';
 } else {
+	webpackConfig.entry.polyfill = [ path.join( __dirname, 'client', 'lib', 'polyfill' ) ];
```

Comment:
i'd probably bump this out to a separate webpack build entirely. This will be affected by things like the vendor dll / vendor chunk, commons plugins, etc. I know it has no deps right now, but if could be a footgun in the future.